client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -116,23 +117,17 @@ float getHardShadow(sampler2D shadowsampler, vec2 smTexCoord, float realDistance
 
 
 #if SHADOW_FILTER == 2
-	#define PCFBOUND 3.5
-	#define PCFSAMPLES 64.0
+	#define PCFBOUND 1.5 // 4x4
```

Comment:
Do we need to make the same changes for the object shader?

Reply:
I want to make all the improvements in the nodes shader first, then transplanting the changes to objects shader is a single commit.